### ISSUE TITLE
Error Running Project on iOS

### ISSUE THREAD

**Issue by jneterer** [Fubinator/react-native-userbase-poc#1]
@Fubinator No need to spend time on your end figuring this out, I'm getting some odd errors trying to get the project up and running. Firstly, not that this is an error, but when running `npm install` the `postinstall` command is logging something interesting:

```> reactnativeuserbasepoc@0.0.1 postinstall /Users/jacobneterer/Desktop/SterlingDS/react-native-userbase-poc
> rn-nodeify --hack --install

not overwriting "assert"
not overwriting "browserify-zlib"
not overwriting "console-browserify"
not overwriting "constants-browserify"
not overwriting "react-native-crypto"
not overwriting "dns.js"
not overwriting "domain-browser"
not overwriting "events"
not overwriting "@tradle/react-native-http"
not overwriting "https-browserify"
not overwriting "react-native-os"
not overwriting "path-browserify"
not overwriting "process"
not overwriting "punycode"
not overwriting "querystring-es3"
not overwriting "react-native-level-fs"
not overwriting "react-native-udp"
not overwriting "stream-browserify"
not overwriting "string_decoder"
not overwriting "timers-browserify"
not overwriting "tty-browserify"
not overwriting "url"
not overwriting "util"
not overwriting "react-native-tcp"
not overwriting "vm-browserify"
not overwriting "readable-stream"
not overwriting "react-native-randombytes"
not reinstalling buffer
not reinstalling inherits
not overwriting shim.js. For the latest version, see rn-nodeify/shim.js
```

Maybe that is expected? When going through your instructions, I couldn't find `/node_modules/parse-asn1/node_modules/pbkdf2/lib/default-encoding.js` to make the one change but I have a feeling its because the `postinstall` command failed. 

Lastly, when running the project in XCode I am getting this error `ld: 144 duplicate symbols for architecture x86_64`. Like I said, if you know what could be the issue off-hand let me know. Otherwise I'll keep researching. Just wanted to post this here other people's reference in case they have a similar issue.

**Comment by Fubinator** [comment]
Uhh interesting, a couple of ideas for the postinstall things:
An idea would be to try running `react-native link`. Even though, in my understanding, react native has autolinking in versions > 0.60, this might be helpful. (See the instructions [here](https://github.com/tradle/react-native-crypto#install)). Another idea would be to remove  the `shim.js` file and run `npm install` (or just the postinstall hook) again.
I'm not sure why you do not get the file, you might be right that it's the postinstall hook not overwriting things. I'm gonna look into those problems after work!

Regarding XCode: I have never played around with XCode and it would be awesome if you could find something out about that issue. I might look into the iOS version, when I have the time to set that up. Maybe this issue helps with that problem: https://github.com/facebook/react-native/issues/23613

**Comment by jneterer** [comment]
@Fubinator How have you been running the project? On an Android simulator? If so, it could be that a dependency wasn't set up right for iOS. I can get the project running on an Android simulator

**Comment by Fubinator** [comment]
@jneterer Yes, I can get it running on an Android simulator as well. What I meant to say is, that I've never played around with iOS simulators and I kinda struggle to set it up on Linux :smile:
